### ISSUE TITLE
Fail to build cli on Ubuntu 14.04

### ISSUE THREAD

**Issue by troydai** [dotnet/cli#1963]
## Steps to reproduce
1. Clone cli repo;
2. Install all the necessary pacakges;
3. `./build.sh`

## Expected  behavior
Build successfully

## Actual behavior
```
Build failed: System.Exception: '.version' file not found in '/home/troydai/code/cli/.dotnet_stage0/Linux/cli' folder
   at Microsoft.DotNet.Cli.Build.PrepareTargets.LocateStage0(BuildTargetContext c)
   at Microsoft.DotNet.Cli.Build.Framework.BuildContext.ExecTarget(BuildTarget target)
```

## Environment data
`dotnet --version` output:

**Comment by troydai** [comment]
Releated https://github.com/dotnet/cli/issues/1955

**Comment by troydai** [comment]
Fixed in build `1.0.0.001897`
